backend/s3/s3.go

```diff
@@ -1968,9 +1970,20 @@ This should be true, false or left unset to use the default for the provider.
 `,
 			Default:  fs.Tristate{},
 			Advanced: true,
-		}, {
-			Name: "use_presigned_request",
-			Help: `Whether to use a presigned request or PutObject for single part uploads
+		},
```

Comment:
We have to reindent lines in this file. But i didn't because this merge has another purpose.